app/controllers/magic_link/sign_up_controller.rb

```diff
@@ -0,0 +1,18 @@
+class MagicLink::SignUpController < ApplicationController
+  def new; end
+
+  def create
+    candidate = Candidate.new(candidates_params)
+    MagicLinkFlow.sign_up(candidate)
+
+    redirect_to check_your_emails_path
```

Comment:
`check_your_emails_path` is a slightly misleading name as it implies another controller, which isn't the implementation any more. I think I'd sooner `redirect_to :show`?

Reply:
We decided to use `render :show` in the sign up controller to remove the need for a check your email route. 
Seems like a nicer way to do it.